src/screens/AdminPortal/ManageTag/ManageTag.module.css

```
@@ -26,12 +26,38 @@
+  display: inline-flex;
+  align-items: center;
+  gap: var(--space-3);
+}
+
 .editButton {
   background-color: var(--color-blue-200);
   color: var(--color-gray-700);
   border-color: var(--color-gray-100);
   --bs-btn-active-bg: var(--color-blue-500);
   --bs-btn-active-border-color: var(--color-gray-100);
+  width: var(--space-14);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)